onnxruntime/test/util/include/test/test_environment.h

```diff
@@ -10,8 +10,12 @@
 #endif
 
 namespace onnxruntime {
+class Environment;
+
 namespace test {
 
+const ::onnxruntime::Environment& GetEnvironment();
+
```

Comment:
nit: Does this need '::onnxruntime::'? Or at least it should never need the '::' prefix at the start.

Reply:
sure